Added swatch support to CS products ref (#110)
AdobeDocs/commerce-webapi: src/pages/_includes/graphql/catalog-service/product-view.md

Field | Data Type | Description
--- | --- | ---

> `addToCartAllowed` | Boolean | A flag stating if the product can be added to the cart.

Reviewer: ```suggestion
`addToCartAllowed` | Boolean | Indicates whether the product can be added to the cart.
```

Field | Data Type | Description
--- | --- | ---
`id` | ID | The ID of an option value.

> `isDefault` | Boolean | States if the option is the default.

Reviewer: ```suggestion
`isDefault` | Boolean | Indicates whether the option is the default.
```

Field | Data Type | Description
--- | --- | ---

> `addToCartAllowed` | Boolean | A flag stating if the product can be added to the cart.

Reviewer: ```suggestion
`addToCartAllowed` | Boolean | Indicates whether the product can be added to the cart.
```
